uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -420,6 +420,10 @@ pub mod filters {
         Ok(as_ct.as_codetype().literal(literal))
     }
 
+    pub fn ffi_type(type_: &impl AsType) -> Result<FfiType, askama::Error> {
+        Ok(type_.as_type().into())
+    }
```

Comment:
I think this could live in https://github.com/mozilla/uniffi-rs/blob/main/uniffi_bindgen/src/backend/filters.rs and not have to be duplicated for each bindings language.